README.md

```diff
@@ -82,5 +82,5 @@ contract MyContract is Adminable, Batch, NoDelegateCall {
 
 This repo is licensed under the [GNU General Public License](/LICENSE-GPL.md) with the below exceptions:
 
-- [`ComptrollerManager.sol`](./src/ComptrollerManager.sol) and [`SablierComptroller.sol`](./src/SablierComptroller.sol)
-  are licensed under the Business Source License 1.1 (`BUSL-1.1`), see [`LICENSE.md`](./LICENSE.md).
+- [`Comptrollerable.sol`](./src/Comptrollerable.sol) and [`SablierComptroller.sol`](./src/SablierComptroller.sol) are
+  licensed under the Business Source License 1.1 (`BUSL-1.1`), see [`LICENSE.md`](./LICENSE.md).
```

Comment:
We don't actually need BUSL for the `Comptroller` and the `Comptrollerable` contracts. They are basic governance contracts.

Reply:
I agree for `Comptrollerable` but the `SablierComptroller` contains oracle and fee related logic which is specific to us, which was the reason behind adding BUSL to it. Let me know if you still want to remove BUSL from the `SablierComptroller`.

Reply:
Oh, OK. Let's keep it `SablierComptroller`, then.

Reply:
https://github.com/sablier-labs/evm-utils/pull/43/commits/5d1566b61c9a78cd10745f66a59a45e44f418665.

I made a few more changes to license files structure. Since, some files are licensed under GPL and some under BUSL, the `package.json` should contain `SEE LICENSE IN < filename >` as per the [npm docs](https://docs.npmjs.com/cli/v11/configuring-npm/package-json). That means the `LICENSE.md` file should be the entry point for all the licensing details.

Reply:
Great finding. Should we do the same in all other repos?